### PR TITLE
Fix some documentations & comments

### PR DESCRIPTION
**Pull request by pajlada**
- **add comments for the new reward filters**
- **slightly improve documentation of r9k values**

### CODE REVIEW

**Review by pajlada — COMMENTED** [review]
image

File: src/widgets/settingspages/GeneralPage.cpp [hunk 1]
```diff
@@ -929,7 +935,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         },
         [](auto args) {
             return fuzzyToInt(args.value, 5);
-        });
+        },
+        true,
+        "A value of 5s means if there's a 5s break inbetween messages, we will "
```
Comment (Mm2PL) [inline src/widgets/settingspages/GeneralPage.cpp:940]: "inbetween" is wrong. IMO this should be:
```suggestion
        "A value of 5s means if there's a 5s break between messages, we will "
```

File: src/widgets/settingspages/GeneralPage.cpp [hunk 2]
```diff
@@ -904,7 +904,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                           toggleLocalr9kShortcut + ".");
     layout.addCheckbox("Hide similar messages", s.similarityEnabled);
     //layout.addCheckbox("Gray out matches", s.colorSimilarDisabled);
-    layout.addCheckbox("By the same user", s.hideSimilarBySameUser);
+    layout.addCheckbox(
+        "By the same user", s.hideSimilarBySameUser, false,
+        "When checked, messages that are very similar to each other can still "
+        "be sent as long as they're sent by different users.");
```
Comment (Mm2PL) [inline src/widgets/settingspages/GeneralPage.cpp:910]: Messages can be shown, not sent

**Review by Mm2PL — CHANGES_REQUESTED** [review]
:tf:

**Review by Mm2PL — APPROVED** [review]

**Pull request merged**